lib/components/bm-confirm-location-on-map.js

```diff
@@ -0,0 +1,148 @@
+/* @flow */
+'use strict'
+
+const geolocation = require('@blinkmobile/geolocation')
+
+/* :: import type {
+  AngularCallbackOptions, Coordinates, NgModelController
+} from '../../types.js' */
+
+const mod = require('../module.js')
+const utils = require('../utils.js')
+
+class BmConfirmLocationOnMapController {
+  /* :: static $inject : string[] */
+
+  // internal use
+
+  /* :: $digest: () => void */
+  /* :: coords: ?Coordinates */
+  /* :: isEditing: boolean */
+
+  // public attributes (after casts / checks)
+
+  /* :: ngDisabled: boolean */
+  /* :: ngModel: NgModelController */
+  /* :: ngReadonly: boolean */
+
+  constructor ($scope /* : Object */) {
+    this.$digest = $scope.$digest.bind($scope)
+    this.coords = null
+    this.isEditing = false
+  }
+
+  $onInit () {}
+
+  $onChanges () {
```

Comment:
im not too fussed and honestly, the object sometimes has this property, sometimes has that property, i dont know if it would be a performance impact with all the branching to check. 

Where it does come in handy is the `isFirstChange` (or something like that) if you need to do something on the first change, which I don't think you need to.

I was just curious about your choice, because it didn't occur to me that I could do it that way